forms/TreeDropdownField.php

```diff
@@ -217,11 +217,12 @@ public function Field($properties = array()) {
 		Requirements::css(FRAMEWORK_DIR . '/thirdparty/jquery-ui-themes/smoothness/jquery-ui.css');
 		Requirements::css(FRAMEWORK_DIR . '/client/dist/styles/TreeDropdownField.css');
 
-		if($this->showSearch) {
-			$emptyTitle = _t('DropdownField.CHOOSESEARCH', '(choose folder)', 'start value of a dropdown');
-		} else {
-			$emptyTitle = _t('DropdownField.CHOOSE', '(Choose)', 'start value of a dropdown');
-		}
+		$item = DataObject::singleton($this->sourceObject);
+		$emptyTitle = _t(
+			'DropdownField.CHOOSE_MODEL',
+			'(Choose {name})',
+			['name' => $item->i18n_singular_name()]
```

Comment:
Did we make a decision on array declaration style? My understanding was to stick with full `array()` syntax